botocore/utils.py

```diff
@@ -3315,3 +3315,79 @@ def _serialize_if_needed(self, value, iso=False):
                 return value.isoformat()
             return value.strftime('%Y-%m-%dT%H:%M:%S%Z')
         return value
+
+
+# This parameter is not part of the public interface and is subject to abrupt
+# breaking changes or removal without prior announcement.
+# Mapping of services that have been renamed for backwards compatibility reasons.
+# Keys are the previous name that should be allowed, values are the transformed
```

Comment:
I'm not sure if we should include service id in this explanation. We should be able to shorten it to something like: "values are the documented/preferred client name" to simplify it down.